src/algebraic_topology/dold_kan/homotopies.lean

```diff
@@ -0,0 +1,204 @@
+/-
+Copyright (c) 2022 Joël Riou. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Joël Riou
+-/
+
+import algebra.homology.homotopy
+import algebraic_topology.dold_kan.notations
+
+/-!
+
+# Construction of homotopies for the Dold-Kan correspondence
+
+(The general strategy of proof of the Dold-Kan correspondence is explained
+in `equivalence.lean`.)
```

Comment:
This doc string refers to files that do not yet exist in Mathlib. I appreciate you are migrating code from another repo and the doc string is great but I worry about what will happen if something interrupts the migration and we end up with references to missing files.

Reply:
If the migration is interrupted, everything in the subdirectory `dold_kan/` would have to be trashed because the main results would not be there and most of the individual files are only a small part of a chain towards these results. I am not sure what is the best procedure here?

Reply:
Perhaps we could just add a short comment along the lines of "TODO (@joelriou) continue adding the various files referenced below" so that if there is an interruption then there is some local explanation to anyone who might be confused in the future.

Reply:
The ideal (perhaps unrealistic) is that module docs get updated along the way, as more stuff arrives in mathlib. In the meantime Oliver's suggestion (to leave some breadcrumb comments, or at least some explanation that the files don't yet exist) is fine.

Reply:
Ok. I have followed Oliver's suggestion.